downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -126,6 +115,23 @@ public function main() returns error? {
 }
 ```
 
+#### `http` package
+
+- Introduced support for server-sent events.
+- Introduced service contract types.
+- Introduced default status code response record type.
```

Comment:
```suggestion
- Introduced the default status code response record type.
```

or

```suggestion
- Introduced default status code response record types.
```

Whichever relevant.

Reply:
First one is the correct one, updated with [069777d](https://github.com/ballerina-platform/ballerina-dev-website/pull/9226/commits/069777dc7b2f70efee7abbadb8f22294170d1ff5)